src/Microsoft.AspNetCore.Rewrite/Internal/ModRewrite/ConditionPatternParser.cs

```diff
@@ -139,7 +138,7 @@ public static ParsedModRewriteInput ParseActionCondition(string condition)
         /// <param name="context"></param>
         /// <param name="invert"></param>
         /// <returns></returns>
-        public static ParsedModRewriteInput ParseProperty(ParserContext context, bool invert)
+        private ParsedModRewriteInput ParseProperty(ParserContext context, bool invert)
```

Comment:
For a static method you can confidently tell it won't change the state of an instance. It guards the expectation. It also helps to improve readability because the reader can easily make this basic assumption.